Proof-reading on tut - PoE (#204)
substrate-developer-hub/substrate-docs: v3/tutorials/02-proof-of-existence/index.mdx

1. When a new proof is added to the blockchain.
2. When a proof is removed.

> Events can contain some additional data. In this case, each event will also display who trigger the

Reviewer: ```suggestion
Events can contain some additional data. In this case, each event will also display who triggers the
```

event (`AccountId`), and the proof data (as `Vec<u8>`) that is being stored or removed. Note that

> convention is to include an array with descriptive names for these parameters at the end of event

Reviewer: ```suggestion
convention is to include an array with descriptive names for these parameters when documenting
```

> documentation.

Reviewer: ```suggestion
an event.
```

yarn start
```

> A new tab should open in your web browser and you'll the following interface.

Reviewer: ```suggestion
A new tab should open in your web browser and you should see the following interface.
```
